src/main/scala/com/sumologic/sumobot/core/Receptionist.scala

```diff
@@ -90,10 +90,10 @@ class Receptionist(rtmClient: SlackRtmClient,
       log.info(s"sending - ${channel.name}: $text")
       rtmClient.sendMessage(channel.id, text)
 
-    case OutgoingImage(channel, imageFile, contentType, title) =>
+    case OutgoingImage(channel, imageFile, contentType, title, commentOpt) =>
```

Comment:
Nit: I'm not generally a fan of repeating the type of a variable in the name of it, so `comment` is sufficient.